libs/test/SmartStoreTest/src/com/salesforce/androidsdk/store/EncryptedSmartStoreTest.java

```diff
@@ -1,39 +0,0 @@
-/*
```

Comment:
There is no real value testing with and without encryption. 
At the very beginning, non-encrypted db was done with plain SQLite and encrypted with SQLCipher but it is no longer the case. So getting rid of EncryptedSmartStoreTest and PlainSmartStoreTest and renaming AbstractSmartStoreTest to SmartStoreTest.